### ISSUE TITLE
Liée Tables 

### ISSUE THREAD

**Issue by Azerosse** [TheShark34/OpenAuth-Server#10]
Bonsoir je viens d'installer OpenAuth server cependant quand je m'inscrit sur mon site ma table openauth_user de ma base donnée  ne se remplit pas mais la table users de Mineweb (j'utilise Mineweb) prend les pseudo et mdp de l'inscription y a t'il un moyen de relier les 2 tables Openauth_users et users (mineweb) ?

**Comment by Vahelnir** [comment]
Il faut adapter l'application pour qu'elle enregistre ce qu'il faut dans la table de Mineweb, ou alors il faut utiliser un plugin pour Mineweb qui te permet de faire ca.

**Comment by Azerosse** [comment]
D'accord je ne crois pas que Mineweb possède un plugin pour ça cependant si j'ai bien compris si je vais dans les fichiers register , validate authenticate.php etc si je change les requète SQL comme celle ci : 'SELECT * FROM openauth_users WHERE accessToken=:accessToken', ['accessToken' => $accessToken]);
en remplaçant openauth_users par users (nom de la table mineweb ) çela devrait fonctionner ?

**Comment by Azerosse** [comment]
Ok erreur de ma part mineweb possède un plugin je vais me renseigner !

**Comment by Vahelnir** [comment]
Je ne pense pas que cela fonctionnera juste en modifiant cela, la façon dont mineweb gère ses tables n'est pas la même que la notre, de plus leur façon de chiffrer les mots de passes non plus.

**Comment by Litarvan** [comment]
@Vahelnir T'es pas forcément au courant j'imagine, mais quelqu'un a développé une integration OpenAuth <-> Mineweb qui est vachement utilisée, et ça remplace OpenAuth Server je crois

**Comment by Azerosse** [comment]
@Litarvan moi non plus j étais pas au courrant est ce que pourrai me donner le nom de cette intégration car avec le plugin mineweb openauth est imcompatible ..

**Comment by Vahelnir** [comment]
Si je sais @Litarvan, mais je ne savais pas que c'était totalement finalisé. J'imagine que c'est celui de EmpireDev -> https://github.com/MineWeb/Plugin-Auth. L'avantage c'est qu'il sera probablement plus à jour que cette version là, tout en étant fait pour mineweb.

**Comment by Azerosse** [comment]
Je crois pas que ça soit cela car le plugin développé pour mineweb est totalement incompatible avec openauth ..

**Comment by Litarvan** [comment]
Oui, mais il te permet de faire ce que tu veux, OpenAuth n'a juste plus d'utilité c'est pour ça qu'il faut l'enlever si tu veux l'utiliser, mais ça s'intègre avec l'OpenLauncherLib etc...
